src/core/router/transports/impl.cc

```diff
@@ -275,6 +274,7 @@ void Transports::SendMessage(
 void Transports::SendMessages(
     const kovri::core::IdentHash& ident,
     const std::vector<std::shared_ptr<kovri::I2NPMessage>>& msgs) {
+  LogPrint(eLogDebug, "Transports: sending messages");
```

Comment:
This was because there are multiple parts in the codebase where multiple messages are being sent and they don't use SendMessage() so there would be no logging of the messages being sent. It is a small thing, but I figured I would fix it. SendMessage() calls the SendMessages() so it would still print the log for those cases.